material/partials/product-switcher.html

```diff
@@ -33,22 +35,32 @@
               </div>
             </a>
           </div>
+          {% endif %} {% if config.product != "traefik-hub" %}
+          <div class="dm-item">
+            <div class="dmi-image hub">
+              <img src="{{ 'assets/images/traefik-hub-logo-white.svg' | url }}" alt="Traefik Hub">
+            </div>
+            <a class="dmi-details" href="https://doc.traefik.io/traefik-hub/">
+              <div class="dmi-title">Traefik Hub</div>
+              <div class="dmi-description">
+                The cloud-native application proxy
```

Comment:
@geraldcroes what should we put below the Traefik Hub name in this dropdown menu?

<img width="493" alt="Screen Shot 2022-10-25 at 00 50 37" src="https://user-images.githubusercontent.com/38889364/197680268-d46bfe3b-8d4c-4b71-9c31-d3e51a0d7d88.png">

@gndz07 @ldez please feel free to update this part once we have the definition and proceed with review/merge.